data/thundercats/ab6a9660-d927-11e6-a0b3-f776e713e0a9.json

```diff
@@ -0,0 +1 @@
+{"name":"lion-o","origin":"thundera","id":"ab6a9660-d927-11e6-a0b3-f776e713e0a9","group":"thundercats"}
```

Comment:
If we similar file saving methods in the future, make sure to add your mock ``data`` directory to your .gitignore by just typing your ``data (or whatever your directory name is)`` in the .gitignore.